_posts/2020-07-16-1.0.0-release.md

```diff
@@ -0,0 +1,211 @@
+`DenseUnionScalar`; ARROW-8866).
+
+Arrow can now be built for iOS using the right set of CMake options, though
+we don't officially support it (ARROW-8795).  See
+[this writeup](https://github.com/UnfoldedInc/deck.gl-native-dependencies/blob/master/docs/iOS-BUILD.md#arrow-v0170)
+for details.
+
+### Compute functions
+
+The compute kernel layer was extensively reworked (ARROW-8792).  It now offers
+a generic function lookup, dispatch and execution mechanism.  Furthermore,
+elaborate internal scaffoldings make it vastly easier to write new function
+kernels.
+
+Several compute functions have been added.  Unicode-compliant predicates and
+transforms, such as lowercase and uppercase transforms, are now available.
+
+The available compute functions are listed exhaustively in the Sphinx-generated
+documentation.
+
+### Datasets
+
+Datasets can now be read from CSV files (ARROW-7759).
+
+### Feather
+
+The Feather format is now available in version 2, which is simply the Arrow
+IPC file format with another name.
+
+### IPC
+
+By default, we now write IPC streams with metadata V5.  However, metadata V4
+can be requested by setting the appropriate member in `IpcWriteOptions`.
+
+V4 as well as V5 metadata IPC streams can be read properly, with one
+exception: a V4 metadata stream containing Union arrays with top-level
+null values will refuse reading.
+
+Support for dictionary replacement and dictionary delta was implemented
+(ARROW-7285).
+
+### Parquet
+
+Writing files with the LZ4 codec is disabled, because it produces files
+incompatible with the widely-used Hadoop Parquet implementation (ARROW-9424).
+Support will be reenabled once we align the LZ4 implementation with the
+special buffer encoding expected by Hadoop.
+
+## C# notes
+
+## Go notes
+
+## Java notes
+
+## JavaScript notes
+
+## Python notes
+
+The size of wheel packages is significantly reduced.  One side effect is
+that these wheels do not enable Gandiva anymore (ARROW-5082).
+
+The Scalar class hierarchy was reworked to more closely follow its C++
+counterpart (ARROW-9017).
+
+TLS CA certificates are looked up more reliably when using the S3 filesystem,
+especially with manylinux wheels (ARROW-9261).
+
+The encoding of CSV files can now be specified explicitly, defaulting to UTF8
+(ARROW-9106).  Custom timestamp parsers can now be used for CSV files
+(ARROW-8711).
+
+Filesystems can now be implemented in pure Python (ARROW-8766).  As a result,
+[fsspec](https://filesystem-spec.readthedocs.io)-based filesystems can now
+be used in datasets (ARROW-9383).
+
+## R notes
+
+The R package added support for converting to and from many additional Arrow types. Tables showing how R types are mapped to Arrow types and vice versa have been added to the [introductory vignette][6], and nearly all types are handled. In addition, R `attributes` like custom classes and metadata are now preserved when converting a `data.frame` to an Arrow Table and are restored when loading them back into R.
```

Comment:
Please, can text be properly word-wrapped at ~80 characters for easier review and editing?